### PR TITLE
eAPI 1.6

### PR DESCRIPTION
**Pull request by jiripudil**
see https://github.com/csob/paymentgateway/wiki/eAPI-1.6
- [x] option to choose which version to use
- [x] changes in `payment/init`
  - `language` is required (not affected)
  - added `ttlSec`, `logoVersion`, and `colorSchemeVersion`
- [x] `payment/oneclick` as a replacement for `payment/recurrent`
- [ ] extensions - might require more research (and honestly is not a priority) - maybe resolve in another pull request so that it does not block this one?
- [x] tag new release? What should be the version number? I'd vote for 2.0 since dropping paymentRecurrent altogether is a massive BC break

### CODE REVIEW

**Comment by jiripudil** [comment]
This is as far as I can get now, I'm moving on to adapt CsobPaygateNette. @fprochazka feel free to review this one :)

File: tests/KdybyTests/CsobPaymentGateway/Client.paymentProcess.phpt [hunk 1]
```diff
@@ -44,7 +40,7 @@ class ClientPaymentProcessTest extends CsobTestCase
 
 		$processResponse = $this->client->paymentProcess($response->getPayId());
 		Assert::type(RedirectResponse::class, $processResponse);
-		Assert::match(Configuration::DEFAULT_SANDBOX_URL . '/payment/process/%A%', $processResponse->getUrl());
+		Assert::match(Configuration::DEFAULT_SANDBOX_URL . '/v%A%/payment/process/%A%', $processResponse->getUrl());
```
Comment (fprochazka) [inline tests/KdybyTests/CsobPaymentGateway/Client.paymentProcess.phpt]: `%a%` should be enough, right?

**Comment by fprochazka** [comment]
>  extensions - maybe resolve in another pull request so that it does not block this one?

Definitely :) #17 

> tag new release? ... I'd vote for 2.0 

:+1:

**Comment by fprochazka** [comment]
And great work @jiripudil!

**Comment by jiripudil** [comment]
@fprochazka okay then, as far as I'm concerned, this PR is ready for merge

**Pull request merged**